views/components/Sidebar/SidebarLink.css

```diff
@@ -0,0 +1,21 @@
+.sidenav_link {
+  text-decoration: none;
+  padding: 6px 12px 6px 12px;
+  display: flex;
+  flex-direction: row;
```

Comment:
`row` is the default direction.

```suggestion
```

Reply:
[465c369](https://github.com/AMP-SCZ/dpdash/pull/458/commits/465c36952e0f8176957c65d89dfea1c6b86ca296)